SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,100 @@
+Supported Operations
+####################
+
+Base Tensors
+============
+
+The protocols use these base tensors for their operations. Some protocol operations are supported by these tensors.
+
+.. table::
+
+    +------------------------+--------------+-----------------------+----------------------+
+    | **Tensor**             | **Protocol** | **Number of parties** | **Shares per party** |
+    +========================+==============+=======================+======================+
+    | ReplicatedSharedTensor | ABY3, Falcon | N                     | N - 1                |
+    +------------------------+--------------+-----------------------+----------------------+
+    | ShareTensor            | FSS, SPDZ    | N                     | 1                    |
+    +------------------------+--------------+-----------------------+----------------------+
+
+
+Base Tensor Operations
+======================
+
+.. table::
+
+    +------------------+----------------+-------------------------+-----------------------+
+    | **Visibility**   | **Operation**  | **Tensor**              | **Number of Parties** |
+    +==================+================+=========================+=======================+
+    | Public & Private | Addition       | - ReplicatedShareTensor | - 3                   |
+    |                  |                | - ShareTensor           | - 2                   |
+    |                  +----------------+-------------------------+-----------------------+
+    |                  | Subtraction    | - ReplicatedShareTensor | - 2+                  |
+    |                  |                | - ShareTensor           | - 2                   |
+    |                  +----------------+-------------------------+-----------------------+
+    |                  | Multiplication | - ReplicatedShareTensor | - 2+                  |
```

Comment:
@danielorihuela Multiplication is only public. Not Public & Private.

Reply:
@kamathhrishi done!